### PR TITLE
Supports adding a test module namespace

### PR DESCRIPTION
**Pull request by abea**
The existing tests didn't support official Apostrophe modules well since they created a symlink test node module named after the package directory. A3 modules do not include the namespace in the package directory typically. This supports adding the namespace for a test module.

In our official modules in packages like `form` or `seo` we would use `testModule: '@apostrophecms'` on test apps to namespace the symlink.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -279,9 +279,17 @@ module.exports = async function(options) {
...
+    let pkgNamespace = '';
+    if (pkgName.includes('/')) {
+      const parts = pkgName.split('/');
+      pkgNamespace = '/' + parts.slice(0, parts.length - 1).join('/');
+    }
+
     if (!fs.existsSync(testDir + '/node_modules')) {
-      fs.mkdirSync(testDir + '/node_modules' + addDirNamespace(options.testModule), { recursive: true });
-      fs.symlinkSync(moduleDir, testDir + '/node_modules' + addDirNamespace(options.testModule) + '/' + require('path').basename(moduleDir), 'dir');
+      fs.mkdirSync(testDir + '/node_modules' + pkgNamespace, { recursive: true });
```
Comment (boutell) [inline index.js:291]: What version of Node did this recursive flag arrive in ?

Comment (boutell) [reply index.js:291]: Otherwise looks good.

**Review by boutell — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (abea) [reply index.js:291]: Added in Node 10.

Comment (boutell) [reply index.js:291]: Unfortunately A2 still runs on some Node 8 servers. 😢

**Review by boutell — CHANGES_REQUESTED** [review]
Unfortunately A2 still runs on some Node 8 servers. 😢

**Comment by abea** [comment]
This isn't for A2.

File: index.js (hunk 1 above)
Comment (abea) [reply index.js:291]: This isn't for A2.

Comment (abea) [reply index.js:291]: I can implement this differently in A2.

**Review by boutell — APPROVED** [review]
Sorry, too many balls in the air.

**Pull request merged**